poetry.lock

```diff
@@ -2002,19 +2002,22 @@ six = [
     {file = "six-1.16.0.tar.gz", hash = "sha256:1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926"},
 ]
 snowflake-connector-python = [
-    {file = "snowflake-connector-python-2.7.6.tar.gz", hash = "sha256:331628e0ca02764492b83b5e47703bd857115ae29346b72de32d7fb7c9ec548b"},
-    {file = "snowflake_connector_python-2.7.6-cp310-cp310-macosx_10_14_universal2.whl", hash = "sha256:4aa90cddbc46f8f580ad78cbb9d4741fd66e665d3223f7f907d859554d2025c2"},
-    {file = "snowflake_connector_python-2.7.6-cp310-cp310-manylinux_2_12_x86_64.manylinux2010_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:ea850b2d816263ae63c00ad60f4df7006ddb1854f28aaa5b3688bf0e5e458dca"},
-    {file = "snowflake_connector_python-2.7.6-cp310-cp310-win_amd64.whl", hash = "sha256:5f3faa237f22d8305c8e48d8b5110c4015485b2cfa683e176fa89eafe57f6a8e"},
-    {file = "snowflake_connector_python-2.7.6-cp37-cp37m-macosx_10_14_x86_64.whl", hash = "sha256:74f55a0d25a53378994939b43a2a55105f12b1b92c8b7cb863b490c9a4243147"},
-    {file = "snowflake_connector_python-2.7.6-cp37-cp37m-manylinux_2_12_x86_64.manylinux2010_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:781473bd60388ce651ad094288f231bcae7fa5cd447e64d12577736cb37b8c96"},
-    {file = "snowflake_connector_python-2.7.6-cp37-cp37m-win_amd64.whl", hash = "sha256:8cfce34bbb910a396b230768658ac65578f28073b62cc9c85289ebbd217e839d"},
-    {file = "snowflake_connector_python-2.7.6-cp38-cp38-macosx_10_14_x86_64.whl", hash = "sha256:e180d285e93d5e3fe2b7b4958212d4964d45f1eca94f03dedc4e569bcc550bfd"},
-    {file = "snowflake_connector_python-2.7.6-cp38-cp38-manylinux_2_12_x86_64.manylinux2010_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:f59ed0c3223548a25aca9bcacf2c5cde04ad5e15366f0cf9e7b20dc34554b1cf"},
-    {file = "snowflake_connector_python-2.7.6-cp38-cp38-win_amd64.whl", hash = "sha256:8c82eda884967ef00d1b7aa59c925c83368f886bfa447c8fa2a81dced03dea70"},
-    {file = "snowflake_connector_python-2.7.6-cp39-cp39-macosx_10_14_universal2.whl", hash = "sha256:22437ee26baaa83cabcebbe9fbc9c95b36bdca441db61822588a622e32bcbecd"},
-    {file = "snowflake_connector_python-2.7.6-cp39-cp39-manylinux_2_12_x86_64.manylinux2010_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:8dbaf2dfed6c7ba4fd90c9f0ec604f86a2e070e4219c182f79f44ca225749c18"},
-    {file = "snowflake_connector_python-2.7.6-cp39-cp39-win_amd64.whl", hash = "sha256:d94eea3047c6cf6d5e4bb8a4b9f961f77f87e25671e7cb6257c0459419a9b194"},
+    {file = "snowflake-connector-python-2.7.8.tar.gz", hash = "sha256:9cfb07b048bcb1fe646d03f7480ccb6a877e9c45067302e90bcff7fd72f6bc2f"},
```

Comment:
Ooof. Check out line 1872. For some reason the pyjwt entry isn't getting updated there, and I have no idea why. :(

Reply:
Oh good catch, bummer :( I did poetry install with a fresh env to test this and did get version 2.3.0 like this says

Reply:
I did `poetry update` and then `poetry install` in a fresh env and that worked!

Reply:
check out my latest commit and lmk what you think